controllers/api.controllers.js

```diff
@@ -0,0 +1,7 @@
+// require in JSON folder
```

Comment:
Please delete any comments that you have left in your code please! Keep it tidy 😄